server/server.ts

```diff
@@ -0,0 +1,41 @@
+// @ts-ignore
```

Comment:
I think remove the `@ts-ignore`. No errors on my local, sounds like an intellj skill issue 😄.

Reply:
Guess I will just live with the redlines 😢